Update public-access.md (#7165)
umbraco/UmbracoDocs: umbraco-cloud/set-up/project-settings/public-access.md

When enabled only team members on the project and users whose IPs have been allowed, can access the frontend of the project.

All environments on Umbraco Cloud projects can be protected by Public access. It requires you to enter your Cloud credentials in order to view the frontend.

{% hint style="info" %}

> By default, we add **Basic Authentication** to all trials and non-live environments. 

Reviewer: ```suggestion
By default, **Basic Authentication** is enabled on all trial and non-live environments. 
```
